src/plugins/data/public/ui/dataset_selector/configurator.tsx

```diff
@@ -51,6 +52,17 @@ export const Configurator = ({
     return languages[0];
   });
 
+  const shouldDisplayTimeFieldSelector = (ds: Dataset, lang: string, fields: DatasetField[]) => {
```

Comment:
```suggestion
  const shouldDisplayTimeFieldSelector = (ds: Dataset, lang: string, timeFields: DatasetField[]) => {
```
nit: reads easier. Had to go lower in the code to see if you were actually only passing time fields to make sure that you were passing the correct list

Reply:
Irrelevant if you accept my suggested logic